spec/rubocop/cop/lint/numeric_operation_with_constant_result_spec.rb

```diff
@@ -23,17 +23,6 @@
     RUBY
   end
 
-  it 'registers an offense when the modulo of variable and 1 is taken' do
-    expect_offense(<<~RUBY)
-      x % 1
-      ^^^^^ Numeric operation with a constant result detected.
-    RUBY
-
-    expect_correction(<<~RUBY)
-      0
-    RUBY
-  end
```

Comment:
To prevent it from becoming an incorrect behavior again, can you replace `expect_offense` with `expect_no_offenses` instead of removing the test?

Reply:
Sure, done 👍🏻